derived_images/portus/README.md

```diff
@@ -87,51 +70,77 @@ by the upstream project.
-extension.
+uses TLS to secure itself. The Registry certificate must be placed inside
+of `/certificates`, the `init` script of this image will automatically import
+it if it ends with the `.crt` extension.
 
 ### Logging
 
-Both Portus' and Apache's messages are redirected to `stdout` and `stderr`. This
-makes it possible to handle the logs of this image in the usual ways.
+All logging is done to `stdout` and `stderr`. This makes it possible to handle
+the logs of this image in the usual ways.
 
 ## Deployment
 
 It's possible to deploy this image using one of the existing orchestration
-solutions for Docker images.
+solutions for Docker images. You can read some examples in the `examples`
+directory of Portus' source code.
 
-However we recommend the usage of [kubernetes](http://kubernetes.io/). We will
-soon release a reference implementation.
+### Executing the crono script
+
+Portus uses [crono](https://github.com/plashchynski/crono) to handle some
+background jobs. You can also execute this piece with this image. In order to do
+this, you need to set the `PORTUS_INIT_COMMAND` environment variable to
+"bin/crono".
 
 ### Environment variables
 
-The following environment variables must be defined to run the image:
+Here's the full list of environment variables:
 
 Security related settings:
 
   * `PORTUS_SECRET_KEY_BASE`: you can generate it using `openssl rand -hex 64`
-  * `PORTUS_PORTUS_PASSWORD`: you can generate it using `openssl rand -hex 64`
+  * `PORTUS_KEY_PATH`: the path of the certificate key. This is the key that
+    Portus will use for the authentication with your Docker registry.
+  * `PORTUS_PASSWORD`: the password of the hidden `portus` user. You can
+    generate it using `openssl rand -hex 64`
+  * `PORTUS_PUMA_TLS_KEY`: The TLS key to be picked by Puma.
+  * `PORTUS_PUMA_TLS_CERT`: The TLS certificate to be picked by Puma.
+  * `PORTUS_CHECK_SSL_USAGE_ENABLED`: Set this to `false` if you want to disable
+    SSL altogether.
 
 Database releated settings:
 
-  * `MARIADB_SERVICE_HOST`: the host running the MariaDB database.
-  * `MARIADB_USER`: the database user to be used.
-  * `MARIADB_PASSWORD`: the password of the database user.
-  * `MARIADB_DATABASE`: the name of the Portus database.
+  * `PORTUS_PRODUCTION_HOST`: the host running the MariaDB database.
+  * `PORTUS_PRODUCTION_USERNAME`: the database user to be used.
+  * `PORTUS_PRODUCTION_PASSWORD`: the password of the database user.
+  * `PORTUS_PRODUCTION_DATABASE`: the name of the Portus database.
 
 Deployment related settings:
 
   * `PORTUS_MACHINE_FQDN_VALUE`: this is the fully qualified domain name of your
     Portus instance (eg: `portus.example.com`).
+
+Some fine tuning for Puma:
+
+  * `PORTUS_PUMA_WORKERS`: the amount of Puma workers to be spawned. Defaults to 1.
+  * `PORTUS_PUMA_MAX_THREADS`: the maximum amount of Puma threads to be
+    created. Defaults to 1.
+  * `RAILS_SERVE_STATIC_FILES`: set this to `true` if you want Puma to serve the
```

Comment:
Yes for two reasons:

- The crono job should be executed in another container, see SUSE/Portus#1254
- Tuning `PORTUS_PUMA_WORKERS` and `PORTUS_PUMA_MAX_THREADS` should avoid any blocking issues.